client.go

```diff
@@ -896,7 +910,7 @@ func (c *Client) closeSession(ctx context.Context, s *Session) error {
 // DetachSession removes the session from the client without closing it. The
 // caller is responsible to close or re-activate the session. If the client
 // does not have an active session the function returns no error.
-func (c *Client) DetachSession(ctx context.Context) (*Session, error) {
+func (c *Client) DetachSession(_ context.Context) (*Session, error) {
```

Comment:
Please leave `ctx` in there since it is part of the documentation.

Reply:
got a unused warning as it is not used, therefore removed it. Reverting it...

Reply:
Same comment about over-aggressive linter settings from people coming from other languages.